airbyte-webapp/src/components/ui/Toast/Toast.module.scss

```diff
@@ -1,3 +1,89 @@
+@use "scss/colors";
+@use "scss/variables" as vars;
+
+$toast-icon-size: 13px;
+$toast-icon-container-size: 34px;
+
+@keyframes slide-up-animations {
+  0% {
+    transform: translate(-50%, -100%);
+    bottom: -49px;
+  }
+
+  100% {
+    transform: translate(-50%, 0);
+    bottom: 49px;
+  }
+}
+
+@mixin type($name, $color, $background) {
+  &.#{$name} {
+    background-color: $background;
+    border: 1px solid $color;
+
+    .iconContainer {
+      background-color: $color;
+    }
+
+    .toastIcon {
+      color: $color;
+    }
+  }
+}
+
+.toastContainer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: vars.$spacing-md;
+  position: fixed;
+  box-sizing: border-box;
+  bottom: 49px;
+  left: 50%;
+  transform: translate(-50%, 0);
+  z-index: 20;
```

Comment:
This value should be part of the `z-indices.css` file.

Reply:
fixed